blog/zk-snark-pallet.md

```diff
@@ -260,4 +260,4 @@ Now we can verify if we received a `VerificationSuccess` event. To do it, we nee
 As you see, verification succeed, and the event was emitted from the Aclie account.
 
 ## Summary
-Thanks to blockchain technology and zk-SNARKs, Alice proves that she knows the solution to the Bob puzzle without revealing it. Everything was stored on the blockchain, so the result of the contest was fully transparent for everyone.
+Thanks to blockchain technology and zk-SNARKs, Alice proved that she known the solution to the Bob puzzle without revealing it. Everything was stored on the blockchain, so the result of the contest was fully transparent for everyone.
```

Comment:
"Had" is missing? Alice proved that she had known. I guess "Alice proved that she knew" would be fine too. :)